### DISCUSSION TITLE
Bforartists 3 version 3.2.0

### DISCUSSION THREAD

**Discussion by ReinerBforartists** [Bforartists/Bforartists#3046] [General]
Hello Friends,

Blender has released a new version, Blender 3.2.0, and so we release a new version of Bforartists too. Bforartists 3 Version 3.2.0 is a maintenance release to keep Bforartists updated with the latest changes from Blender 3.2.

The binary downloads are attached. But can as usual also be found in the download section of the official webpage. https://www.bforartists.de/download/
And the release note is here: https://www.bforartists.de/release-notes/

Many thanks to the team and everybody involved.

Reiner

![bfa_320](https://user-images.githubusercontent.com/13919702/172891200-473aacc6-2ee5-4e2d-908b-687f00312bdf.jpg)

<hr /><em>This discussion was created from the release <a href='https://github.com/Bforartists/Bforartists/releases/tag/v3.2.0'>Bforartists 3 version 3.2.0</a>.</em>

**Comment by Istiak-VR-dev** [comment]
Nice . 
can u tell me how to turn off the right click object context menu in blender??

**Comment by ReinerBforartists** [comment]
Thanks Kronos568 :)

This happens in the keymap. Search for entries for the right mouse. And everything with context in the title. Or simply search for context menu.

Kind regards
Reiner 

![prefs](https://user-images.githubusercontent.com/13919702/173800336-19aad96d-fe82-41e0-8ac2-e8f2498f35ff.jpg)

**Reply by Istiak-VR-dev** [reply]
yeah . i am  deselecting that but still it is showing in the viewport

**Comment by Istiak-VR-dev** [comment]
![Screenshot (26)](https://user-images.githubusercontent.com/106044051/173803245-776c4263-b60a-47e9-86cd-1fbd86db89f8.png)

**Comment by ReinerBforartists** [comment]
I have unfortunately no idea what goes on at your end. But here this works fine. Odd.

![objectcontext](https://user-images.githubusercontent.com/13919702/173805690-755289b1-4a49-4513-8e1e-ac74d79a8880.jpg)

**Reply by Istiak-VR-dev** [reply]
thanks for the reply

**Reply by Istiak-VR-dev** [reply]
bforartist is awesome
